src/styles/card.scss

```diff
@@ -0,0 +1,114 @@
+.card {
```

Comment:
fix size of the card: 

![image](https://github.com/user-attachments/assets/af64c0e2-4434-49c2-9509-2940c1c15cc2)